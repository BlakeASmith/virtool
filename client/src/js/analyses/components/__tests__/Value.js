import AnalysisValue from "../Value";

describe("<AnalysisValue />", () => {
    it("should render", () => {
        const props = {
            color: "red",
            label: "foo",
            value: 1312
        };
        const wrapper = shallow(<AnalysisValue {...props} />);
        expect(wrapper).toMatchSnapshot();
    });

    it("should render without props", () => {
        const wrapper = shallow(<AnalysisValue />);
        expect(wrapper).toMatchSnapshot();
    });
});
