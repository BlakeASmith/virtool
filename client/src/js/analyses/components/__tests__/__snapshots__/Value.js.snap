// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AnalysisValue /> should render 1`] = `
<Value__AnalysisValue>
  <Value__AnalysisValueNumber
    color="red"
  >
    1312
  </Value__AnalysisValueNumber>
  <Value__AnalysisValueLabel>
    foo
  </Value__AnalysisValueLabel>
</Value__AnalysisValue>
`;

exports[`<AnalysisValue /> should render without props 1`] = `
<Value__AnalysisValue>
  <Value__AnalysisValueNumber />
  <Value__AnalysisValueLabel />
</Value__AnalysisValue>
`;
